### PR TITLE
Fix about adding plugin with alias

### LINKED ISSUE #218: [BUG]: Wrong behavior about plugin alias and something to be improved
**Describe the bug**
 1. 以相同别名安装不同插件后，会下载插件，但是安装失败
预期：不下载插件
2. 以别名安装某插件后，可以直接安装或使用其他别名安装该插件；但是直接安装某插件以后，无法再用别名安装该插件
预期：以不同名称添加的插件应视作不同插件，只要别名不与已安装插件名称重复即可安装
3. 对旧版本插件名称格式`<category>/<plugin-name>`的兼容仅对官方网址生效
预期：与网址无关，应对任何网址生效

### PR DESCRIPTION
**Pull request by yanecc**
Fixes #218

### CODE REVIEW

**Comment by yanecc** [comment]
Suspend for https://github.com/version-fox/vfox/issues/218#issuecomment-2067923309

**Comment by yanecc** [comment]
Fixes #220

File: internal/manager.go [hunk 1]
```diff
@@ -384,22 +385,24 @@ func (m *Manager) downloadPlugin(downloadUrl string) (string, error) {
 }
 
 func (m *Manager) Add(pluginName, url, alias string) error {
+	// For compatibility with older versions of plugin names <category>/<plugin-name>
+	if strings.Contains(pluginName, "/") {
```
Comment (aooohan) [inline internal/manager.go:389]: While this is unlikely to be the case with custom plugins, it's harmless enough to keep.

**Review by aooohan — APPROVED** [review]
LGTM.

This change, weakens the name of the plugin itself. It's based on the indexed repository and the alias name entered by the user. I think it's good.

**Pull request merged**